diff --git a/node_modules/probot-scheduler/index.js b/node_modules/probot-scheduler/index.js
index b3070c0..4df70d1 100644
--- a/node_modules/probot-scheduler/index.js
+++ b/node_modules/probot-scheduler/index.js
@@ -94,9 +94,9 @@ module.exports = (app, options) => {
     const github = await app.auth(installation.id)
 
     const repositories = await github.paginate(
-      github.apps.listRepos.endpoint.merge({ per_page: 100 }),
+      github.apps.listReposAccessibleToInstallation.endpoint.merge({ per_page: 100 }),
       response => {
-        return response.data.repositories
+        return response.data
       }
     )
 
